changelogs/fragments/sd_wan_support.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Comment:
Not sure if this is bugfixes or minor_changes.

Reply:
yeah it's a feature, so I'll change it to `minor_changes`